### ISSUE TITLE
The xml file aliases-config-pfSense.xml is missing starting tag.

### ISSUE THREAD

**Issue by thomasha** [freedomofpress/securedrop#2277]
# Bug

## Description

The xml file [aliases-config-pfSense.xml](https://github.com/freedomofpress/securedrop/blob/develop/install_files/network_firewall/aliases-config-pfSense.xml) is missing the starting tag.

The file starts with 
`aliases>` should start with `<aliases>`

**Comment by thomasha** [comment]
There is also an mistyping in the [interfaces](https://github.com/freedomofpress/securedrop/blob/develop/install_files/network_firewall/interfaces-config-pfSense.xml)  file, if following the guide, the subnet on OPT1 is set to 32, but should be 24. 

```
<opt1>
    <if>igb2</if>
    <descr>OPT1</descr>
    <enable/>
    <ipaddr>10.20.2.1</ipaddr>
    <subnet>32</subnet>
    <spoofmac/>
</opt1>
```

**Comment by redshiftzero** [comment]
Good catches @thomasha, will fix. Let us know if you spy anything else off with the templates.

**Comment by redshiftzero** [comment]
Relevant: #2282. Added to 0.4.4 milestone.

**Comment by redshiftzero** [comment]
Actually, we should still use the firewall templates as a reference, so I went ahead and fixed this in #2283.

**Comment by redshiftzero** [comment]
Fixed by #2283
